apps/auth/src/pages/index.tsx

```diff
@@ -1,14 +1,18 @@
 import Footer from '@klicker-uzh/shared-components/src/Footer'
-import { Button, H1 } from '@uzh-bf/design-system'
+import { Button, Checkbox, H1 } from '@uzh-bf/design-system'
 import { signIn, signOut, useSession } from 'next-auth/react'
 import Image from 'next/image'
 import { useRouter } from 'next/router'
 
+import useStickyState from 'src/hooks/useStickyState'
+
 function SignInOutButton() {
   const router = useRouter()
 
   const { data: session } = useSession()
 
+  const [tosChecked, setTosChecked] = useStickyState('tos-agreement', false)
```

Comment:
use sticky state -> ensure the ToS do not have to be accepted on every single login, but only once per browser (store accept in local storage)